packages/webdriver/src/utils.js

```diff
@@ -22,8 +22,14 @@ export function isSuccessfulResponse (statusCode, body) {
      * response contains a body
      */
     if (!body || typeof body.value === 'undefined') {
-        log.debug('request failed due to missing body')
-        return false
+        if (body && body.status === 0) {
+            /**
+            * A workaround for WinAppDriver. For some messages, WinAppDriver doesn't return value when status is 0.
```

Comment:
This should be fixed in WinAppDriver then. WebdriverIO follows the W3C standard and so should the driver.